qa/pull-tester/rpc-tests.py

```diff
@@ -198,30 +176,52 @@ def runtests():
     if args.help:
         # Print help for rpc-tests.py, then print help of the first script and exit.
         parser.print_help()
-        subprocess.check_call((RPC_TESTS_DIR + test_list[0]).split() + ['-h'])
+        subprocess.check_call((config["environment"]["SRCDIR"] + '/qa/rpc-tests/' + test_list[0]).split() + ['-h'])
+        sys.exit(0)
+
+    if len(test_list) == 0:
+        print("No rpc tests to run. Scripts not found:")
+        print(tests)
         sys.exit(0)
 
-    coverage = None
+    runtests(test_list, config["environment"]["SRCDIR"], config["environment"]["BUILDDIR"], config["environment"]["EXEEXT"], args.parallel, args.coverage, passon_args)
```

Comment:
Maybe stupid question...not sure how main() can call runtests() without having declared it yet.